syncsweptsine.py

```diff
@@ -368,6 +368,12 @@ def time(self):
         self._update()
         return self._time
 
+    @property
+    def phi(self):
+        """Returns the sweep argument."""

```

Comment:
It is the argument of the *sin* function. I suggest:
"""Returns the instantaneous phase."""